### DISCUSSION TITLE
point of creation of community containers

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3505] [Questions] [answered]
Are community containers created before or after the domain verification? Since otherwise, I don't understand how the caddy community container should work

**Comment by szaimen** [comment, marked as answer]
It gets created after the domain verification.

The trick is the following:

https://github.com/nextcloud/all-in-one/blob/7336337c2ee522756583eb11f97c369f3abda108/community-containers/caddy/caddy.json#L41-L42

So basically you should follow the "normal" instructions but add the community caddy container. Then after the domain was validated it adjusts the settings automatically and caddy in the community container takes over.

**Reply by Zoey2936** [reply]
so it launches in normal mode for verification and then switches to reverse proxy mode while caddy starts?

**Reply by szaimen** [reply]
> so it launches in normal mode for verification and then switches to reverse proxy mode while caddy starts?

Basically yes
